guides/common/modules/con_smartproxy-performance-tests.adoc

```diff
@@ -25,6 +26,7 @@ For more precise numbers, see the table below.
 
 When we compared download performance from {ProjectServer} vs. from {SmartProxyServer}, we have seen only about 5% speedup, but that is expected as {SmartProxyServer}'s main benefit is in getting content closer to geographically distributed clients (or clients in different networks) and in handling part of the load {ProjectServer} would have to handle itself.
 In some smaller hardware configurations (8 CPUs and 24 GiB), {ProjectServer} was not able to handle downloads from more than 500 concurrent clients, while a {SmartProxyServer} with the same hardware configuration was able to service more than 1000 and possibly even more.
+endif::[]
 
 .Concurrent registrations use case
```

Comment:
This whole section is very Katello specific IMHO. Only on Katello you have Apache running a reverse proxy.

Reply:
I think for now this is good enough.